apps/api-server/src/routes/api/project.js

```diff
@@ -302,9 +302,24 @@ router.route('/:projectId') //(\\d+)
     req.pendingMessages = [{ key: `project-${project.id}-update`, value: 'event' }];
     if (req.body.url && req.body.url != project.url) req.pendingMessages.push({ key: `project-urls-update`, value: 'event' });
 
+    // Update allowedDomains if creating a new site
+    let updateBody = req.body;
+    if((project?.config?.widgets?.allowedDomains || []).length === 0 && req?.body?.url){
+      updateBody.config = updateBody.config || {};
+      updateBody.config.widgets = updateBody.config.widgets || {};
+      updateBody.config.widgets.allowedDomains = [req.body.url];
+
+      // Update client (auth-db)
+      let adminAuthConfig = await authSettings.config({ project: project, useAuth: 'openstad' });
+      service.updateClient({
+        authConfig: adminAuthConfig,
+        project: updateBody
+      })
```

Comment:
De `useAuth: 'openstad'` is geen gegeven. Het is ook mogelijk dat iemand inlogt met een andere auth dan `openstad`, en dat wordt nu overschreven in principe.

Zou je hier nog een check willen bouwen of er geen andere authSettings gebruikt wordt op het project?

Reply:
Heb deze weggehaald, dan zou hij nu de project config default moeten pakken.